src/repositories/game-repository.js

```diff
@@ -16,29 +17,35 @@ const Game = t.createModel('Game', {
   status: t.type.string().enum(Object.values(GAME_STATUSES)).default(GAME_STATUSES.UNDEFINED),
   createdBy: t.type.string(),
   closedBy: t.type.string(),
+  bigBuyIn: t.type.number(),
+  smallBuyIn: t.type.number(),
   participants: t.type.array().schema(t.type.object().schema({
     playerId: t.type.string(),
     turnOrder: t.type.number(),
+    seat: t.type.string().enum(Object.values(PARTICIPANT_SEATS)).default(PARTICIPANT_SEATS.UNDEFINED),
     isCurrentTurn: t.type.boolean(),
-    participationStatus: t.type.string().enum(Object.values(PARTICIPATION_STATUSES)).default(PARTICIPATION_STATUSES.UNDEFINED),
+    participationStatus: t.type.string().enum(Object.values(PARTICIPATION_STATUSES))
+      .default(PARTICIPATION_STATUSES.UNDEFINED),
     placing: t.type.number(),
     chips: t.type.array().schema(t.type.object().schema({
       chipId: t.type.string(),
       amount: t.type.number()
     }))
-  })),
+  }).options({ enforce_extra: 'remove' })),
   pot: t.type.array().schema(t.type.object().schema({
     chipId: t.type.string(),
     amount: t.type.number()
   })),
-  createdAt: t.type.date().default(r.now()),
+  createdAt: t.type.date().default(Date.now),
```

Comment:
Vi skickar inte in datum som genereras clientside väl? För i sådana fall måste vi tänka på tidszonerna. Var kör vi för tid serverside och hur ska vi konvertera tidszon clientside?

Reply:
Mycket sant. UTC här och än så länge visar vi inte tiden någonstans så kanske inte behöver göra något än